src/scss/components/_modal.scss

```diff
@@ -0,0 +1,106 @@
+@use 'abstracts' as *;
+
+.modal {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 20;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 500px;
+  background: $color__white;
+  box-shadow: 0 6px 8px 0 rgba(164, 164, 164, 0.5);
+  // This animation is here as a stop-gap measure, but the modal component should be adjusted so it doesn't get destroyed when the close button is clicked. Instead, it should have a class applied that hides it off screen.
+  animation: slide-in 0.6s forwards;
+  animation-timing-function: ease-in-out;
+
+  &__header,
+  &__footer,
+  &__content {
+    padding-right: rem(32px);
+    padding-left: rem(32px);
+  }
+
+  &__header,
+  &__footer {
+    display: flex;
+    flex: 1 1 auto;
+    min-height: rem(108px);
+    padding-top: rem(32px);
+    padding-bottom: rem(32px);
+  }
+
+  &__header {
+    position: relative;
+    justify-content: space-between;
+    align-items: flex-end;
+    background-color: $color__highlight-cell;
+    border-bottom: 1px solid rgba($color__gray-400, 0.24);
```

Comment:
Not sure if you and Tucker talked about this, but it's `$color__gray-460` in the mockup

Reply:
Good catch! I actually adjusted this because although the color is `$color__gray-460`, it also has an opacity of 0.4 applied to it. Since we can't adjust the opacity on just the border, I did the math of what `$color__gray-460` * 0.4 opacity would be.

Reply:
You're so smart :)